src/lib/IPFS/ipfs.js

```diff
@@ -0,0 +1,109 @@
+/*!
+ * Copyright(c) ConsenSys Software Inc.
+ * Copyright(c) https://consensys.net/
+ * MIT Licensed
+ */
+/* eslint-disable */
+import { ERROR_LOG, errorLogger } from '../error/handler.js';
+import fs from 'fs';
+import { create as ipfsClient, globSource } from 'ipfs-http-client';
+import { toBase64 } from '../utils.js';
+
+export default class IPFS {
+  #ipfsClient;
```

Comment:
IMO this one should be public so everyone could use the ipfsClient on their own.